### PR TITLE
nauty: correct location of nauty*.pc files

### PR DESCRIPTION
**Pull request by dimpase**
correct the mistake in #159841 where we assumed that "make install" does the right thing with *.pc files, but in fact they go into a wrong location.

Upstream agrees that it's a work item for them (via email, as there is no bug tracker).

- [x ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/n/nauty.rb [hunk 1]
```diff
@@ -61,3 +65,17 @@ def install
...
++++ b/makefile.in
+@@ -17,7 +17,7 @@ exec_prefix=@exec_prefix@
+ bindir=@bindir@
+ libdir=@libdir@
+ includedir=@includedir@
+-pkgconfigexecdir=${prefix}/libdata/pkgconfig
++pkgconfigexecdir=${libdir}/pkgconfig
+ 
+ INSTALL=@INSTALL@
+ INSTALL_DATA=@INSTALL_DATA@
```
Comment (chenrui333) [inline Formula/n/nauty.rb:81]: does this get sent to the upstream?

**Comment by chenrui333** [comment]
@dimpase you can run `brew style --fix nauty` to fix the style issue.

**Comment by dimpase** [comment]
I see in CI
```
[Correctable] FormulaAudit/ComponentsOrder: livecheck (line 13) should be put before patch (line 11)
    livecheck do ...
    ^^^^^^^^^^^^
```
will fix

**Comment by dimpase** [comment]
> @dimpase you can run `brew style --fix nauty` to fix the style issue.

locally, on my machine, it passed...

File: Formula/n/nauty.rb (hunk 1 above)
Comment (dimpase) [reply Formula/n/nauty.rb:81]: Yes, I have suggested this patch to upstream.
(Upstream does not have a public VCS repo, so I described it in prose)

Comment (dimpase) [reply Formula/n/nauty.rb:81]: I'll email upstream with a link to this PR

**Comment by dimpase** [comment]
Maybe I am bad at reading the manual, but where should `patch` go?
I don't see anything on this in https://docs.brew.sh/Formula-Cookbook#patches

Now I see
```
 /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/n/nauty.rb:21:3: C: [Correctable] FormulaAudit/ComponentsOrder: bottle (line 21) should be put before patch (line 19)
    bottle do ...
    ^^^^^^^^^
```

**Review by carlocab — APPROVED** [review]

**Pull request merged**